fe/fe-core/src/main/java/org/apache/doris/statistics/StatsRecursiveDerive.java

```diff
@@ -38,11 +36,11 @@ private static class Inner {
      * This parameter is an input and output parameter,
      * which will store the derivation result of statistical information in the corresponding node
      */
-    public void statsRecursiveDerive(PlanNode node) throws UserException {
```

Comment:
statsRecursiveDerive's parameter should not change to PlanStats, since Nereids will never use this to do stats Derive